manual/manual.tex

```diff
@@ -273,13 +273,20 @@ \section{Runtime Configuration\label{runtime-config}}
 \begin{lstlisting}
 interface RuntimeOpts {
   estimator?: "velocity" | "reservoir" | "exact" | "countdown",
-  yieldInterval?: number /* must be greater than zero */
+  yieldInterval?: number /* must be greater than zero */,
+
+  stackSize?: number /* must be greater than zero */
+  restoreFrames?: number /* must be greater than zero */,
 }
 \end{lstlisting}
 
-These options control how frequently Stopify yields control to the browser
+The first two options control how frequently Stopify yields control to the browser
 (\lstinline|yieldInterval|) and the mechanism that it uses to determine
-elapsed time (\lstinline|estimator|).
+elapsed time (\lstinline|estimator|). The last two options tell Stopify how deep
+the browser stack is and how many frames should be restored onto the JS stack
```

Comment:
Can you add `(\lstinline|stackSize|)` and `(\lstinline|restoreFrames|)` just to be consistent and explicit like in the previous sentence?